packager/packager.cc

```diff
@@ -1127,6 +1128,29 @@ struct PackageSpecBodyWalk {
                 }
             }
         }
+
+        if (send.fun == core::Names::layer()) {
+            if (info.layer.has_value()) {
+                if (auto e = ctx.beginError(send.loc, core::errors::Packager::InvalidLayer)) {
+                    e.setHeader("Repeated declaration of `{}`", send.fun.show(ctx));
+                    e.addErrorLine(ctx.locAt(info.layer.value().second), "Previous declaration found here");
```

Comment:
```suggestion
                    e.addErrorLine(ctx.locAt(info.layer.value().second), "Previously declared here");
```

A little consistence with other errors. We only use `"declaration found here"` once but we use `"declared here"`/`"defined here"` many times.